apps/web/utils/plurals/PolishPlurals.tsx

```diff
@@ -0,0 +1,28 @@
+export const OpinionPolishPlurals = (value: number) => {
+	const formatter = new Intl.PluralRules('pl', {
+		type: 'cardinal',
+	});
+
+	const polishPluralsPosts = {
+		zero: 'opinii',
+		one: 'opinia',
+		two: 'opinie',
+		few: 'opinie',
+		many: 'opinii',
+		other: 'opinie',
+	};
+
+	const plural = formatter.select(value);
+	const pluralForm = polishPluralsPosts[plural];
+
+	return `${value} ${pluralForm}`;
+};
+
+export const TimePolishPlurals = (
```

Comment:
Dlaczego jedne nazwy funkcji zczynają się wielką literą, a inne małą?

Reply:
ta powinna być z małej? chodzi o to że np: jest handleSubmit?